src/main/java/com/microsoft/aad/msal4j/AadInstanceDiscoveryProvider.java

```diff
@@ -129,34 +129,59 @@ private static AadInstanceDiscoveryResponse sendInstanceDiscoveryRequest(URL aut
                                                                              MsalRequest msalRequest,
                                                                              ServiceBundle serviceBundle) {
 
-        String region = StringHelper.EMPTY_STRING;
         IHttpResponse httpResponse = null;
-
-        //If the autoDetectRegion parameter in the request is set, attempt to discover the region
-        if (msalRequest.application().autoDetectRegion()) {
-            region = discoverRegion(msalRequest, serviceBundle);
+        String providedRegion = msalRequest.application().azureRegion();
+        String detectedRegion = null;
+        int regionOutcome = 0;
```

Comment:
should be renamed to reflect usage of variable as telemetry datapoint

Reply:
Good point, renamed to `regionOutcomeTelemetryValue` in the latest commit.